tests/default_vars.sh

```diff
@@ -560,6 +561,8 @@ export FNALBC="'global_snowfree_albedo.bosu.t126.384.190.rg.grb',"
 export FNVETC="'global_vegtype.igbp.t126.384.190.rg.grb',"
 export FNSOTC="'global_soiltype.statsgo.t126.384.190.rg.grb',"
 export FNSMCC="'global_soilmgldas.t126.384.190.grb',"
```

Comment:
Do we need both FNSMCC and FNSMCC_control?

Reply:
Yes at this time. The non-control related tests are using FNSMCC. I hope we can unify them when all the global atm tests are updated.